sem_covid/entrypoints/etl_dags/eurlex_covid19.py

```diff
@@ -29,6 +29,7 @@
 FAILURE_KEY = 'failure_reason'
 RESOURCE_FILE_PREFIX = 'res/'
 TIKA_FILE_PREFIX = 'tika/'
+CONTENT_LANGUAGE = "Tika detected language"
```

Comment:
let's call this "language", and let's use the same key in the other DAGS as well.